Duplicati/GUI/Duplicati.GUI.TrayIcon/HttpServerConnection.cs

```diff
@@ -259,6 +259,9 @@ private string GetAuthToken()
             var sha256 = System.Security.Cryptography.SHA256.Create();
             var password = m_password;
 
+            if (m_password == null)

```

Comment:
null is sufficient. Main reason was to not pass null there System.Text.Encoding.UTF8.GetBytes.
If password is "", it will go ok whole function, of course server will return unauthorized.
IsNullOrWhiteSpace will work too.